src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/StringExtentions.cs

```diff
@@ -10,77 +12,207 @@ namespace Exadel.CrazyPrice.Common.Extentions
     /// </summary>
     public static class StringExtentions
     {
+        /// <summary>
+        /// Gets DateTime from a string with format "dd.MM.yyyy HH:mm:ss".
+        /// </summary>
+        /// <param name="stringDate"></param>
+        /// <returns></returns>
+        public static DateTime GetDateTimeInvariant(this string stringDate)
+        {
+            if (string.IsNullOrEmpty(stringDate))
+            {
+                throw new ArgumentNullException(nameof(stringDate));
+            }
+
+            return DateTime.ParseExact(stringDate, "dd.MM.yyyy HH:mm:ss", CultureInfo.InvariantCulture);
+        }
+
+        /// <summary>
+        /// Gets true if string is valid WorkingDays otherwise false.
+        /// </summary>
+        /// <param name="stringWorkingDays"></param>
+        /// <returns></returns>
+        public static bool IsValidWorkingDays(this string stringWorkingDays)
+        {
+            if (string.IsNullOrEmpty(stringWorkingDays) || stringWorkingDays.Length != 7)
+            {
+                return false;
+            }
+
+            var chars = new[] { '0', '1' };
+            return stringWorkingDays.All(stringWorkingDay => chars.Contains(stringWorkingDay));
+        }
+
+        /// <summary>
+        /// Gets string what two and more chars replaced by some one.
+        /// </summary>
+        /// <param name="s"></param>
+        /// <param name="chars"></param>
+        /// <returns></returns>
+        public static string ReplaceTwoAndMoreCharsBySomeOne(this string s, string chars)
+        {
+            if (string.IsNullOrEmpty(chars))
+            {
+                throw new ArgumentNullException(nameof(chars));
+            }
+
+            return s.ReplaceTwoAndMoreCharsBySomeOne(chars.ToCharArray());
+        }
 
         /// <summary>
-        /// Removes 2 and more space. Trim.
+        /// Gets string what two and more chars replaced by some one.
         /// </summary>
         /// <param name="s"></param>
+        /// <param name="chars"></param>
         /// <returns></returns>
-        public static string ReplaceTwoAndMoreSpaceByOne(this string s)
+        public static string ReplaceTwoAndMoreCharsBySomeOne(this string s, char[] chars)
```

Comment:
Probably string to replace? Do we need this method?

Reply:
Used by GetValidContent.